src/main/java/de/tum/in/www1/artemis/service/StaticCodeAnalysisService.java

```diff
@@ -46,16 +46,7 @@ public void createDefaultCategories(ProgrammingExercise programmingExercise) {
         }
 
         // Create new static code analysis using the default configuration as a template
```

Comment:
I think we could get rid of this commend by renaming `newCategories`, e.g. to `newCategoriesWithDefaultConfiguration` - but I guess this is a matter of taste 🤷‍♂️